src/@types/options.ts

```diff
@@ -91,6 +91,12 @@ export interface IOptions {
    */
   ttl?: number | TTLFactory;
 
+  /**
+   * "Session Table Name"
+   * defines session table name. Defaults to sessions
+   */
+  sessionModelName?: Exclude<M, `$${string}`>;
```

Comment:
@jeremygottfried, @wSedlacek , can you help me understand how generics are working here? Specifically, I'm not following the `$${string}`. (Thanks!)

Reply:
The `$${string}` here is looking for any string that starts with a `$`, ie `$use` or `$on` which are used in Prisma to denote internal methods.

The reason it is used here is to make sure the `IPrisma` interface does not infer that `$use` (and the other internal methods) should have a `.create()` method.

Reply:
Ah - got it. Thanks.